include/oneapi/dpl/pstl/histogram_impl.h

```diff
@@ -18,7 +18,10 @@
 
 #include "histogram_extension_defs.h"
 #include "histogram_binhash_utils.h"
+#include "execution_impl.h"
 #include "iterator_impl.h"
+#include "algorithm_impl.h"
+#include "parallel_backend_utils.h"
```

Comment:
I do not see anything from `oneapi::dpl::__utils` used in the code; looks like an artifact of some experimental code.

Reply:
You are correct,  removed.